modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/manager/AddressContactInfoManager.java

```diff
@@ -0,0 +1,127 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.account.admin.web.internal.manager;
+
+import com.liferay.portal.kernel.model.Address;
+import com.liferay.portal.kernel.service.AddressLocalService;
+import com.liferay.portal.kernel.service.AddressService;
+import com.liferay.portal.kernel.service.ServiceContext;
+import com.liferay.portal.kernel.util.ParamUtil;
+import com.liferay.portal.kernel.util.Validator;
+
+import java.util.List;
+
+import javax.portlet.ActionRequest;
+
+/**
+ * @author Danny Situ
+ */
+public class AddressContactInfoManager extends BaseContactInfoManager<Address> {
+
+	public AddressContactInfoManager(
+		String className, long classPK, AddressLocalService addressLocalService,
+		AddressService addressService) {
+
+		_className = className;
+		_classPK = classPK;
+		_addressLocalService = addressLocalService;
+		_addressService = addressService;
+	}
+
+	@Override
+	protected Address construct(ActionRequest actionRequest) throws Exception {
+		String street1 = ParamUtil.getString(actionRequest, "addressStreet1");
+		String street2 = ParamUtil.getString(actionRequest, "addressStreet2");
+		String street3 = ParamUtil.getString(actionRequest, "addressStreet3");
+		String city = ParamUtil.getString(actionRequest, "addressCity");
+		String zip = ParamUtil.getString(actionRequest, "addressZip");
+		long countryId = ParamUtil.getLong(actionRequest, "addressCountryId");
```

Comment:
either you sort them or you name them as the params

Reply:
I vote for the same name as the params